jha/src/router/routes.js

```diff
@@ -2,6 +2,26 @@ export const enrolmentRoutes = {
   HOME_PAGE: {
```

Comment:
You have removed HOME_PAGE elsewhere, should it be removed here as well

Reply:
It's still referenced by other pages, so we need to keep the definition for now.